src/App.jsx

```diff
@@ -14,57 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodText, setGoodText] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodText.length ? null : `No goods selected`}
+      </h1>
```

Comment:
```suggestion
      <h1 className="title is-flex is-align-items-center">
        {goodText.length ? `${goodText} is selected` : 'No goods selected'}
        {goodText.length && (
          <button
            data-cy="ClearButton"
            type="button"
            className="delete ml-3"
            onClick={removeGood}
          />
        )}
      </h1>
```

Reply:
As you can see I added remove good handler function so please move the logic which is related to removing good inside of it.